files/default/sql/ddl/updates/1.4.0.sql

```diff
@@ -21,3 +21,7 @@ UPDATE `hopsworks`.`jupyter_settings` `j`
 JOIN `hopsworks`.`project` `p`
 ON `j`.`project_id` = `p`.`id`
 SET `j`.`base_dir` = CONCAT('/Projects/',`p`.`projectname`,'/Jupyter');
+
+ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN machine_type;
+ALTER TABLE `hopsworks`.`python_dep` DROP COLUMN machine_type;
+ALTER TABLE `hopsworks`.`hosts` DROP COLUMN conda_enabled;
```

Comment:
in `recipe/default.rb` we still have some code related to conda/non-conda hosts. could you please remove it as well?

Reply:
We are using those variables in older dml files. I don't know what the correct approach for purging that code is. https://github.com/logicalclocks/hopsworks-chef/blob/49f7e7c1e54bb298ea2993e67d4f589456ce2e3b/recipes/default.rb#L308